index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
```

Comment:
- [ ] Please the logo should be a link to the home page, if you check websites like youtube and facebook you will notice that their logo sends the user to the home page.